File: Assignment39_5.c
#include <stdio.h>

///////////////////////////////////////////////////////////////////////////////
// Function Name : Display
// Description   : Display lowercase characters from a to f using recursion
// Input         : ---
// Output        : a  b  c  d  e  f
///////////////////////////////////////////////////////////////////////////////
void Display()
{
    static char ch = 'a';

    if(ch <= 'f')             // Base condition
    {
        printf("%c  ", ch);
        ch++; 
        Display();           // Recursive call
    }
}

int main()
{
    Display();
    return 0;
}

File: Assignment39_5.java
///////////////////////////////////////////////////////////////////////////////
// Function Name : Display
// Description   : Display lowercase characters from a to f using recursion
// Input         : ---
// Output        : a  b  c  d  e  f
// Author        : Raj Samir Jadhav
// Date          : 03/12/2025
///////////////////////////////////////////////////////////////////////////////
class Pattern
{
    static char ch = 'a';

    void Display()
    {
        if(ch <= 'f')
        {
            System.out.print(ch + "  ");
            ch++;
            Display();            // Recursive call
        }
    }
}

class Assignment39_5
{
    public static void main(String[] args)
    {
        Pattern obj = new Pattern();
        obj.Display();
    }
}
